### PR TITLE
Use upstream mysql helm chart in the examples

### PR DESCRIPTION
**Pull request by PrasadG193**
## Change Overview
This PR:
- Updates mysql helm chart for kanister-mysql to the latest stable version (Chart link: https://github.com/helm/charts/tree/8f6192fdb70032986dda893e419d34508391853b)
- Adds steps to take data backup and restore in README
- Fixes default values for helm chart in README
 

## Pull request type

Please check the type of change your PR introduces:
- [ ] Work in Progress
- [x] Refactoring (no functional changes, no api changes)
- [ ] Trival/Minor
- [ ] Bugfix
- [ ] Feature
- [x] Documentation

## Test Plan

Tested data backup and restore on S3 as per the steps documented in README

- [x] Manual
- [ ] Unit test
- [ ] E2E

### CODE REVIEW

File: examples/helm/kanister/kanister-mysql/README.md [hunk 1]
```diff
@@ -20,7 +20,12 @@ For basic installation, you can install using the provided Helm chart that will
 Prior to install you will need to have the Kanister Helm repository added to your local setup.
 
 ```bash
-$ helm repo add kanister http://charts.kanister.io
+$ helm repo add kanister https://charts.kanister.io
+```
+You can use following command to install Kanister if not installed already
+
+```bash
+$ helm install --name kanister --namespace kasten-io kanister/kanister-operator --set image.tag=0.20.0
```
Comment (pavannd1) [inline examples/helm/kanister/kanister-mysql/README.md]: [minor] Let's make this namespace `kanister`

File: examples/helm/kanister/kanister-mysql/README.md [hunk 2]
```diff
@@ -61,12 +66,150 @@ $ helm install kanister/kanister-mysql -n my-release --namespace mysql-test \
...
+1 row in set (0.00 sec)
+
+```
+
+## Protect the Application
+
+You can now take a backup of the MySQL data using an ActionSet defining backup for this application. Create an ActionSet in the same namespace as the controller.
+
+```bash
+$ kanctl create actionset --action backup --namespace kasten-io --blueprint my-release-kanister-mysql-blueprint --deployment mysql-test/my-release-kanister-mysql --profile mysql-test/mysql-test-profile
```
Comment (pavannd1) [inline examples/helm/kanister/kanister-mysql/README.md]: Also change the namespace to `kanister` here and everywhere below.

File: examples/helm/kanister/kanister-mysql/README.md [hunk 3]
```diff
@@ -61,12 +66,150 @@ $ helm install kanister/kanister-mysql -n my-release --namespace mysql-test \
...
+
+```bash
+$ kanctl --namespace kasten-io create actionset --action restore --from "backup-llfb8"
+actionset restore-backup-62vxm-2hdsz created
+
+# View the status of the ActionSet
+kubectl --namespace kasten-io describe actionset restore-backup-62vxm-2hdsz
+```
+
+You should now see that the data has been successfully restored to MySQL!
```
Comment (pavannd1) [inline examples/helm/kanister/kanister-mysql/README.md]: Should we say - `Once the ActionSet status is set to "complete", you can see that the data has been successfully restored to MySQL` just to make sure we wait until all the phases are completed?

File: examples/helm/kanister/kanister-mysql/.helmignore [hunk 4]
```diff
@@ -1 +1,2 @@
 .git
+OWNERS
```
Comment (SupriyaKasten) [inline examples/helm/kanister/kanister-mysql/.helmignore]: nit: add new line

File: examples/helm/kanister/kanister-mysql/OWNERS [hunk 5]
```diff
@@ -0,0 +1,4 @@
+approvers:
+- olemarkus
+reviewers:
+- olemarkus
```
Comment (SupriyaKasten) [inline examples/helm/kanister/kanister-mysql/OWNERS]: nit: add new line

File: examples/helm/kanister/kanister-mysql/README.md (hunk 1 above)
Comment (SupriyaKasten) [reply examples/helm/kanister/kanister-mysql/README.md]: @pavannd1 will the e2e pass if we change this to ` kanister ` ?

File: examples/helm/kanister/kanister-mysql/.helmignore (hunk 4 above)
Comment (PrasadG193) [reply examples/helm/kanister/kanister-mysql/.helmignore]: @SupriyaKasten I didn't get you. What is the change you are suggesting?

Comment (ntolia) [reply examples/helm/kanister/kanister-mysql/.helmignore]: I believe she wanted you to add a new line to the end of the file.

File: examples/helm/kanister/kanister-mysql/README.md (hunk 1 above)
Comment (PrasadG193) [reply examples/helm/kanister/kanister-mysql/README.md]: @pavannd1 in the latest commit in values.yaml, we are changing default namespace to `kanister-io` https://github.com/kanisterio/kanister/commit/5e5a18eae9a63f85060a8816bf07577c18bba88f

File: examples/helm/kanister/kanister-mysql/.helmignore (hunk 4 above)
Comment (PrasadG193) [reply examples/helm/kanister/kanister-mysql/.helmignore]: got it, thanks!

File: examples/helm/kanister/kanister-mysql/README.md (hunk 3 above)
Comment (PrasadG193) [reply examples/helm/kanister/kanister-mysql/README.md]: yes, will change this

File: examples/helm/kanister/kanister-mysql/kanister/mysql-blueprint.yaml [hunk 6]
```diff
@@ -1,68 +1,70 @@
...
+        - bash
+        - -o
+        - errexit
+        - -o
+        - pipefail
+        - -c
+        - |
+          s3_path="{{ .ArtifactsIn.mysqlCloudDump.KeyValue.path }}"
+          export AWS_SECRET_ACCESS_KEY="{{ .Profile.Credential.KeyPair.Secret }}"
+          export AWS_ACCESS_KEY_ID="{{ .Profile.Credential.KeyPair.ID }}"
```
Comment (SupriyaKasten) [inline examples/helm/kanister/kanister-mysql/kanister/mysql-blueprint.yaml]: @PrasadG193 please remove the two exports, we do not need them anymore.

File: examples/helm/kanister/kanister-mysql/kanister/mysql-blueprint.yaml [hunk 7]
```diff
@@ -1,68 +1,70 @@
...
+        - bash
+        - -o
+        - errexit
+        - -o
+        - pipefail
+        - -c
+        - |
+          s3_path="/mysql-backups/${MYSQL_NAMESPACE}/${MYSQL_RELEASE}-mysql/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}/dump.sql.gz"
+          mysqldump -u root --password="${MYSQL_ROOT_PASSWORD}" -h ${MYSQL_RELEASE}-mysql --single-transaction --all-databases | gzip - | kando location push --profile '{{ toJson .Profile }}' --path ${s3_path} -
+  delete:
```
Comment (SupriyaKasten) [inline examples/helm/kanister/kanister-mysql/kanister/mysql-blueprint.yaml]: Also just a minor change request: although the order does not matter, just to maintain consistency across the blueprints, can we have actions as backup-restore-delete?

Comment (PrasadG193) [reply examples/helm/kanister/kanister-mysql/kanister/mysql-blueprint.yaml]: Yes, I noticed that after raising the PR. I'll push those changes in the next revision. Thanks

File: examples/helm/kanister/kanister-mysql/kanister/mysql-blueprint.yaml (hunk 6 above)
Comment (PrasadG193) [reply examples/helm/kanister/kanister-mysql/kanister/mysql-blueprint.yaml]: Okay

**Comment by SupriyaKasten** [comment]
@PrasadG193 please update the version to `0.21.0` from `0.20.0` everywhere in this PR.

**Comment by PrasadG193** [comment]
@SupriyaKasten done!

**Comment by tdmanv** [comment]
will merge after the next release

**Pull request closed without merging**